frontend_components/src/encord_active_components/frontend/src/components/explorer/api.ts

```diff
@@ -140,7 +141,7 @@ export const getApi = (
           `${baseUrl}/projects/${projectName}/items/${encodeURIComponent(id)}`
         )
       ).json();
-      return ItemSchema.parse({ ...item, url: `${baseUrl}/${url}` }) as Item;
+      return ItemSchema.parse({ ...item, url }) as Item;
```

Comment:
[nitpick] 💅

Since we aren't transforming the URL we don't need to destructure it

```suggestion
      return ItemSchema.parse(item);
```

and also the statement above to 
```ts
const item = await ...
```

(would have made as part of the previous suggestion but GH is not allowing suggestions on lines that aren't in the diff)